extensions-contrib/influx-extensions/src/main/antlr4/io/druid/data/input/influx/InfluxLineProtocol.g4

```diff
@@ -1,3 +1,21 @@
+/*
```

Comment:
I went and double-checked this file. It was contributed in #5440 by @njhartwell, and it looks like he authored this file and intended to contribute it. So I think we are good to add this header.